app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -109,6 +110,9 @@ open class MainActivity : AppCompatActivity(),
                         "    <span class=\"second last\"></span>" +
                         "</div>" +
                         "<br>"
+        private val GUTENBERG_CODE_BLOCK = "<!-- wp:core/image {\"id\":316} -->\n" +
+                "<figure class=\"wp-block-image\"><img src=\"https://bia.is/hola-que-tal.jpg\" alt=\"\" /></figure>\n" +
```

Comment:
May worth to point to a picture that doesn't return 404?

Reply:
Oops. Changed it.